.travis.yml

```diff
@@ -1,8 +1,9 @@
-dist: trusty
+dist: xenial
 language: python
 python:
   - "3.5"
   - "3.6"
+  - "3.7"
```

Comment:
please also change the deploy job condition... and let's extend `distributions` key to be `sdist bdist_wheel`